tutorials/javaFxPart4.md

```diff
@@ -109,6 +109,28 @@ We will get to that later.
 
    ![Settings for DialogBox](images/javafx/DialogBoxController.png)
 
+<box type="info" seamless>
+
+  Using the FXML files above, text overrun may occur when the text to be displayed exceeds the size of the label, causing the text to not be fully displayed and ending with `...` instead.
+
+  To fix this issue, we can implement one of the suggested changes below to the FXML files:
+
+  <b>1. Changing MainWindow.fxml</b>
+  
+  Remove `prefHeight` for `VBox`:
+  ```xml
+  <VBox fx:id="dialogContainer" prefWidth="388.0" />
+  ```
```

Comment:
Maybe an explanation like:
```markdown
This modification removes the preferred height setting for the VBox container. By not specifying a prefHeight, the VBox is allowed to expand vertically as needed based on its contents. This helps accommodate labels that might need more vertical space due to wrapped text, preventing the text from being cut off and ending with ....
```
Would be appropriate so that students know the impacts of the changes they are making.